### ISSUE TITLE
Heroku push warnig

### ISSUE THREAD

**Issue by xcarpentier** [jontewks/puppeteer-heroku-buildpack#8]
Hello,

Great buildpack, thanks ! :)

I have see a small warning:

`remote: W: --force-yes is deprecated, use one of the options starting with --allow instead.`

Best regard,
X

**Comment by jontewks** [comment]
Thanks for sending this my way. I took that line right from the heroku created chrome buildpack, and its still that way, so I'll have to spend a bit of time and find out how to change it and not break everything!

**Comment by stereobooster** [comment]
Related https://github.com/heroku/heroku-buildpack-google-chrome/issues/25

**Comment by jontewks** [comment]
@stereobooster thanks! I'll be keeping my eye on that. Haven't had much free time to fix it on my own with work and the holidays.

**Comment by inkornate** [comment]
Please keep us informed when this gets resolved. For now this is to be ignored i guess. 
Just FYI, when I opened a ticket on Heroku on this they mentioned it is my application issue.

**Comment by jontewks** [comment]
@inkornate yep, definitely something to fix here. Figured it was safer to follow the official buildpack, especially since this isn't causing anyone an actual issue other than seeing a log message they don't want to see. Could always submit a PR! Perhaps I will have the time this weekend.

**Comment by reinink** [comment]
There has been little activity on in the official Heroku library. 😞

**Comment by jontewks** [comment]
Still keeping an eye on this, there is a PR open to fix it and they are talking about how to make it work for all the different stacks heroku has available: https://github.com/heroku/heroku-buildpack-google-chrome/pull/49

**Comment by jontewks** [comment]
@xcarpentier @inkornate Are you still interested in getting this resolved? I realized I can now make a non-production tag that could be optionally installed but not the default and we could test out making sure the new flags will work correctly. I doubt that anyone has any apps running on cedar-14 anyways, but there is no way for me to be sure I won't break it for some of the people.

I'd love to be able to test this on a heroku-16 stack if either of you have an app on that stack, as I can't create on on it any longer and have verified its working on heroku-18

**Comment by inkornate** [comment]
Jon,Sorry for the delayed reply. I have no sense of urgency here and will leave it to your decision. thanks.

    On Thursday, September 20, 2018, 12:23:20 PM CDT, Jon Tewksbury <notifications@github.com> wrote:  
 
 
Closed #8.
